### PR TITLE
Added Homepage

### PR DESCRIPTION
**Pull request by Mohitk391**

### CODE REVIEW

**Review by JuvitaSaini09 — APPROVED** [review]
Every thing looks good to me

File: style.css [hunk 1]
```diff
@@ -0,0 +1,110 @@
+:root{
+    --white: #ffffff;
+    --black: #000000;
+    --grey: #808080;
+    --off-white: #e7e7e7;
```
Comment (bhtibrewal) [inline style.css:5]: Suggestion: give better naming to color variables, these generic names like black, white are not a good practice

File: style.css [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+    padding: 0 1rem;
+    background-color: var(--grey);
+    border: none;
+    border-radius: 2rem;
+    color: var(--white);
+}
+
+.nav-bar-icon {
+    font-size: 1.5rem;
+    color: black;
```
Comment (bhtibrewal) [inline style.css:48]: Suggestion: maintain consistency use variables for color everywhere

File: style.css [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+}
+
+.main {
+    background-color: var(--off-white);
+    width:max-width;
```
Comment (bhtibrewal) [inline style.css:53]: Why have you used this?

File: style.css [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+    height: fit-content;
```
Comment (bhtibrewal) [inline style.css:54]: Change: this is by default no need to do it

**Review by bhtibrewal — CHANGES_REQUESTED** [review]
You haven't used your component library in this project? it was a necessary criterion
I have written a few everything else looks good to me

File: style.css [hunk 5]
```diff
@@ -0,0 +1,108 @@
...
+    padding: 0 1rem;
+    background-color: var(--clr-grey);
+    border: none;
+    border-radius: 2rem;
+    color: var(--clr-white);
+}
+
+.nav-bar-icon {
+    font-size: 1.5rem;
+    color: var(--clr-black);
```
Comment (anasm11) [inline style.css:48]: names of color can be more generalised, it will be helpful when you intend to change them.

**Review by anasm11 — APPROVED** [review]
suggestion: -more padding can be given to header 
       - you can make banner photo a bit brighter
 Rest looks good.

**Pull request merged**